src/rust/grapl-crypto/src/aead/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod xchacha_blake3;
```

Comment:
Might be good to add a module-level doc string saying what an AEAD is for folks that may not be familiar.